fastly/client.go

```diff
@@ -298,7 +298,7 @@ func (c *Client) RequestJSONAPIBulk(verb, p string, i interface{}, ro *RequestOp
 	if ro.Headers == nil {
 		ro.Headers = make(map[string]string)
 	}
-	ro.Headers["Content-Type"] = jsonapi.MediaType
+	ro.Headers["Content-Type"] = jsonapi.MediaType + "; ext=bulk"
```

Comment:
@bernabas I just added the bulk param to content-type. I didn't remove it from Accept. I'm waiting to hear back in the Moonbeam PR how we want to handle that before I make that change.